### PR TITLE
Add '--' in front of "configure"

### PR DESCRIPTION
**Pull request by peterkappus**
Added -- to create "--configure" in the readme so s3cmd recognises it as a switch.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,4 @@ FROM ubuntu
 
 RUN apt-get update && apt-get install -y s3cmd
 
-VOLUME ["/.s3cmd"]
```
Comment (frioux) [inline Dockerfile:5]: Why remove the volume?

File: README.md [hunk 2]
```diff
@@ -3,34 +3,24 @@
 ## Usage
 
 ```
-mkdir ~/.s3cfg
-
-docker run -it --rm         \
-  --volume ~/.s3cfg:/.s3cfg \
-  --user $(id -u)           \
```
Comment (frioux) [inline README.md:10]: Running in --user mode is to help prevent bugs in docker from allowing a root escape.  Probably not a huge deal but why remove it?

File: README.md [hunk 3]
```diff
@@ -3,34 +3,24 @@
...
-  frew/s3cmd                \
-  configure
-
-docker run -it --rm         \
-  --volume ~/.s3cfg:/.s3cfg \
```
Comment (frioux) [inline README.md:15]: I personally feel safer mounting this separately from the rest, but maybe that's silly (if $pwd is ~ it's moot, right?)

File: README.md [hunk 4]
```diff
@@ -3,34 +3,24 @@
...
+#setup
+docker run -it --rm -v $(pwd):/root frew/s3cmd  --configure
+
+#syncing
+docker run -it --rm -v $(pwd):/root -v $(pwd)/public:/public frew/s3cmd  sync public/ s3://mah-bukkit
 ```
 
 ## Description
 
-The packaged versions of [s3cmd](http://s3tools.org/) tend to be a little out of
```
Comment (frioux) [inline README.md:24]: Why'd you remove my newlines?! ;)

**Review by frioux — COMMENTED** [review]
Look over my thoughts and let me know what you think

File: README.md (hunk 4 above)
Comment (peterkappus) [reply README.md:24]: ;) Good question... probably laziness so it's faster to copy/paste but you're right, it's more legible with them in! :)

File: README.md (hunk 3 above)
Comment (peterkappus) [reply README.md:15]: True...

File: README.md (hunk 2 above)
Comment (peterkappus) [reply README.md:10]: Ah, I didn't know what that did so I pulled it out 😬 Probably sensible to add it back in...

File: Dockerfile (hunk 1 above)
Comment (peterkappus) [reply Dockerfile:5]: I removed this initially so I could play with mounting different volumes thinking that was part of the permission denied issue... but I don't think it is.

**Comment by peterkappus** [comment]
Let me make a few more tweaks based on your (very constructive) comments. Thanks for your input!

File: Dockerfile (hunk 1 above)
Comment (peterkappus) [reply Dockerfile:5]: Also (I remember now) the only way I could get it to work without throwing errors was to let it write to /root/.s3cfg and mount /root to ~/ in the host. There's probably a better way...

**Pull request closed without merging**